store/store.go

```diff
@@ -596,12 +599,27 @@ func (s *Store) setStoreID(r *http.Request) (customStore bool) {
 		}
 	}
 	if storeID != "" {
-		r.Header.Set("X-Ubuntu-Store", storeID)
+		r.Header.Set(hdrSnapDeviceStore[apiLevel], storeID)
 		return true
 	}
 	return false
 }
 
+type apiLevel int
+
+const (
+	apiV1 apiLevel = 0
+	apiV2 apiLevel = 1
```

Comment:
That's quite misleading. Why is that not V0 and V1 respectively, or V1 = 1 and V2 = 2?